source/ios/AdaptiveCards/ADCIOSFramework/ADCIOSFramework/ACRContentStackView.h

```diff
@@ -0,0 +1,22 @@
+//
+//  ACRContentStackView
+//  ACRContentStackView.h
+//
+//  Copyright © 2017 Microsoft. All rights reserved.
+//
+
+#import <UIKit/UIKit.h>
+
+@interface ACRContentStackView: UIView
```

Comment:
The reason is because UIStackView does not have struts to hold itself. It needs outside frame that impose some dimensions and constraints to it.

---
In reply to: [128110253](https://github.com/Microsoft/AdaptiveCards/pull/488#discussion_r128110253) [](ancestors = 128110253)